tests/src/unit-readme.cpp

```diff
@@ -105,11 +105,11 @@ TEST_CASE("README" * doctest::skip())
 
             // or even nicer with a raw string literal
             auto j2 = R"(
-          {
-            "happy": true,
-            "pi": 3.141
-          }
-        )"_json;
+                {
```

Comment:
This is a curious whitespace change.

Reply:
I think I just indented it to bring the closing `)` to the correct level. Makes sense that it was off given you just pasted it from the README.